### PR TITLE
Consolidate terminology sections, seprated by local and cited terms.

### PR DESCRIPTION
**Pull request by gkellogg**
Note that this re-numbers sections, and log points that include the section numbers needed to be updated.

For #62. Doesn't close it, as there are changes requried in RDF Concepts and N-Quads that will update terms.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/pull/71.html" title="Last updated on Jan 24, 2023, 10:13 PM UTC (b6bf3e2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/71/ddc404e...b6bf3e2.html" title="Last updated on Jan 24, 2023, 10:13 PM UTC (b6bf3e2)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -279,61 +279,127 @@ <h2>Typographical conventions</h2>
...
+      <dd>An identifier issuer is used to issue new <a>blank node identifier</a>. It
+        maintains a
+        <a href="#bn-issuer-state">blank node identifier issuer state</a>.</dd>
+      <dt><dfn>hash</dfn></dt>
+      <dd>The lowercase, hexadecimal representation of a message digest.</dd>
+      <dt><dfn>hash algorithm</dfn></dt>
+      <dd>The hash algorithm used by <a>URDNA2015</a>, namely, SHA-256.</dd>
+      <dt><dfn data-lt="mention">mention set</dfn></dt>
+      <dd>The set of all <a>quads</a> in a <a>dataset</a>
+        that mention a node <var>n</var> is called its <a>mention set</a> of <var>n</var>,
```
Comment (iherman) [inline spec/index.html:306]: the term "mention", though intuitive, is not really defined. Either it should be separately defined, or use here something "The set of all <a>quads</a> in a <a>dataset</a> that has <var>n<var> as one of its components".

("component" is used in, e.g., https://www.w3.org/TR/rdf11-concepts/#section-triples, so it is o.k. to use that term without an extra definition.)

File: spec/index.html [hunk 2]
```diff
@@ -279,61 +279,127 @@ <h2>Typographical conventions</h2>
...
-    </dd>
-  </dl>
+  <section id="canon-terms">
+    <h3>Terms defined by this specification</h3>
+    <dl data-sort="ascending">
+      <dt><dfn data-lt="input dataset|input datasets">input dataset</dfn></dt>
+      <dd>The abstract <a>RDF dataset</a> that is provided as input to
+        the algorithm.</dd>
+      <dt><dfn>normalized dataset</dfn></dt>
+      <dd>The immutable, abstract <a>RDF dataset</a> and set of normalized
```
Comment (iherman) [inline spec/index.html:289]:
```suggestion
      <dd>The immutable, abstract <a>RDF dataset</a> and a set of normalized
```

**Review by iherman — APPROVED** [review]

File: spec/index.html (hunk 2 above)
Comment (gkellogg) [reply spec/index.html]: Fixed in b6bf3e294447db5b8ba8d94c0cf23706b0997e66.

File: spec/index.html (hunk 1 above)
Comment (gkellogg) [reply spec/index.html]: Addressed in b6bf3e294447db5b8ba8d94c0cf23706b0997e66.

**Pull request merged**